cmd/main.go

```diff
@@ -71,6 +73,7 @@ func main() {
 	flag.StringVar(&transportHost, "transport-host", "amqp:localhost:5672", "The transport bus hostname or service name.")
 	flag.IntVar(&apiPort, "api-port", 8089, "The address the rest api endpoint binds to.")
 	flag.StringVar(&httpEventPublisher, "http-event-publishers", "", "Comma separated address of the publishers available.")
+	flag.StringVar(&amqInitTimeout, "amq-init-timeout", amqInitTimeoutDefault.String(), "The retry duration for establishing an amq connection.")
```

Comment:
this should transport protocol agnostic . You can't have amq specific  flags on sidecar

Reply:
AMQ is going to go away and it not configurable for PTP operator

Reply:
updated to a generic name

Reply:
removed the arg